### PR TITLE
DST Issue is mostly fixed.

### PR DESCRIPTION
**Pull request by jackpmastrangelo**
Hours between slider and search are now consistent. Still have error where on the 11th scrolling the slider to 11pm sets day to the 12th but keeps the day constant.

### CODE REVIEW

File: src/jquery.daterangepicker.js [hunk 1]
```diff
@@ -1343,10 +1343,25 @@
         }
 
         function changeTime(name, date) {
+            var rangeHour = moment(opt[name + 'Time']).format('HH'),
```
Comment (brianeroth) [inline src/jquery.daterangepicker.js]: @jackpmastrangelo These changes look pretty similar to the changes in `setRange`. Can we abstract out into a function?

Comment (jackpmastrangelo) [reply src/jquery.daterangepicker.js]: Yes I can, I'll get on that

**Comment by brianeroth** [comment]
@PatrickMcGrath29 Can you please review and test functionality as well?

**Comment by PatrickMcGrath29** [comment]
Yeah definitely! I'll have time to review and do testing this afternoon

**Pull request merged**